### PR TITLE
Update windows.md

### PR DESCRIPTION
**Pull request by itsthepo**
Specified and made a NOTE to have the customer run the installer as root or administrator

## Purpose of this pull request

Specified and made a NOTE to have the customer run the installer as root or administrator

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/send-data/installed-collectors/windows.md [hunk 1]
```diff
@@ -53,7 +53,10 @@ If necessary, you can use the binary package to install a Collector on MacOS. S
 This method does not support all advanced settings.
 :::
 
-Run the installer on your server with root or Administrator privileges. If you are not logged in as root or Administrator, you might be prompted to reauthenticate to your system when you start the UI Installer.
+:::note Run the installer on your server with root or Administrator privileges. (i.e., Right-Click Run as Administrator on Windows)
+::: 
+
+If you are not logged in as root or Administrator, you might be prompted to reauthenticate to your system when you start the UI Installer.
```
Comment (kimsauce) [inline docs/send-data/installed-collectors/windows.md:59]:
```suggestion
Run the installer on your server with root or Administrator privileges. For Windows, right-click the installer file and select **Run as Administrator**. If you're not logged in as root or Administrator, you may be prompted to reauthenticate when you start the UI Installer.
```

Comment (kimsauce) [reply docs/send-data/installed-collectors/windows.md:59]: @itsthepo , let me know if this sounds ok. Feel free to tweak.

**Review by kimsauce — CHANGES_REQUESTED** [review]
minor fix

File: docs/send-data/installed-collectors/windows.md (hunk 1 above)
Comment (itsthepo) [reply docs/send-data/installed-collectors/windows.md:59]: Looks good!

**Review by itsthepo — COMMENTED** [review]
Looks good!

**Review by kimsauce — APPROVED** [review]

**Pull request merged**